packages/rainbowkit/src/wallets/getDefaultWallets.ts

```diff
@@ -24,6 +25,7 @@ export const getDefaultWallets = ({
       wallets: [
         injectedWallet({ chains }),
         rainbowWallet({ chains }),
+        tallyHoWallet({ chains }),
```

Comment:
We're not going to add anymore wallets to `getDefaultWallets`. This was only meant to be used internally for demos and testing. Moving forward, we'd like the community to publish their own versions of this helper function and include whatever wallets they find best for their users.

Reply:
Makes sense. Removed